soroban/contracts/web_auth/src/lib.rs

```diff
@@ -0,0 +1,62 @@
+#![no_std]
+use soroban_sdk::{
+    contract, contracterror, contractimpl, contracttype, Address, BytesN, Env, Map, String, Symbol,
+};
+
+#[contracttype]
+#[derive(Clone)]
+enum DataKey {
+    Admin,
+}
+
+#[contract]
+pub struct WebAuthContract;
```

Comment:
❓ why is this contract called WebAuth? Can we rename it to SEP45?

Reply:
I'm not familiar with `contractmeta` but I like the idea of providing at least one way to make it explicit that this is a SEP-45 implementation.